lib/generators/rails_performance/install/templates/initializer.rb

```diff
@@ -21,6 +21,10 @@
   # You can ignore endpoints with Rails standard notation controller#action
   # config.ignored_endpoints = ['HomeController#contact']
 
+  # You can ignore request paths by specifying the beginning of the path.
+  # For example, all routes starting with '/admin' can be ignored:
+  # config.ignored_paths = ['/admin']
```

Comment:
please also include it in the readme

the rest of the changes are fine 👍

Reply:
done